Views generic

dotpointer opened the pull request:
Adding and updating information, most notably a view behavior reference list for all behaviors I found used in the views. This is in response to the 'Document view behaviors' todo note.

dotpointer committed:
docs/views/10-generic, adding and updating information.

dotpointer committed:
docs/views/10-generic, adding and updating behavior information.

dotpointer committed:
docs/views/10-generic, adding and updating content.

dotpointer committed:
docs/views/10-generic, adding and updating content regarding behaviors.

dotpointer committed:
docs/views/10-generic, minor text fixes.

dotpointer committed:
docs/views/10-generic, minor text fixes.

JockeCK reviewed (dismissed):
Wow, we have a lot of helper functions. Great work listing them all. Looks good.

Wurper reviewed (dismissed):
Good stuff.

nomego reviewed (changes requested):
This is way too verbose and this kind of "disconnected" documentation for methods will never be updated.

Link to the generated docs for each behavior instead and make sure it's updated in the annotated documentation for each file, see http://localhost:3000/polymer/cz-behaviors/index.html

dotpointer committed:
Application views, View overview, adding see also-links to local documentation and to dos for documentation on Webcomponents.org.

docs/views/10-generic.rst, adding see also links to local documentation and to do for documentation on Webcomponents.org

dotpointer commented:
> This is way too verbose and this kind of "disconnected" documentation for methods will never be updated. Link to the generated docs for each behavior instead and make sure it's updated in the annotated documentation for each file, see http://localhost:3000/polymer/cz-behaviors/index.html

Pull requests has been issued in cosmoz-frontend to update in file documentation:
https://github.com/Neovici/cosmoz-frontend/pull/184
https://github.com/Neovici/cosmoz-frontend/pull/186
https://github.com/Neovici/cosmoz-frontend/pull/187

nomego reviewed (changes requested):
We should still drop the list of methods from this and only maintain the links to the automatic documentation.
We can keep the behavior description though.

dotpointer commented:
> We should still drop the list of methods from this and only maintain the links to the automatic documentation.

Yes, that was my intent to drop the lists and link directly. I have been awaiting review of the automatic documentation to see if I was on the right track, and now as one is merged then I can continue removing the lists.

dotpointer committed:
Application views, View overview, removing method lists for behaviors with local automatic documentation references.

docs/views/10-generic.rst, removing method lists for behaviors with local automatic documentation
